docker/datahub-gms/Dockerfile

```diff
@@ -15,9 +15,9 @@ RUN apk --no-cache --update-cache --available upgrade \
       echo >&2 "Unsupported architecture $(arch)" ; exit 1; \
     fi \
     && apk --no-cache add tar curl openjdk8-jre bash coreutils gcompat \
-    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-runner/9.4.20.v20190813/jetty-runner-9.4.20.v20190813.jar --output jetty-runner.jar \
-    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-jmx/9.4.20.v20190813/jetty-jmx-9.4.20.v20190813.jar --output jetty-jmx.jar \
-    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-util/9.4.20.v20190813/jetty-util-9.4.20.v20190813.jar --output jetty-util.jar \
+    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-runner/9.4.46.v20220331/jetty-runner-9.4.46.v20220331.jar --output jetty-runner.jar \
```

Comment:
How about a similar update in docker/datahub-ingestion/Dockerfile?

Reply:
I can update the comment, but it doesn't actually use it from what I can tell?